src/assets/scripts/client/scope/RadarTargetModel.js

```diff
@@ -303,6 +306,72 @@ export default class RadarTargetModel {
         return [true, 'AMEND ALTITUDE'];
     }
 
+    /**
+     * Generate a string to be used for the first row of a datablock
+     *
+     * @for RadarTargetModel
+     * @method buildDataBlockRowOne
+     * @returns {string}
+     */
+    buildDataBlockRowOne() {
+        let dataBlockRowOne = `${this.aircraftModel.callsign}`;
+
+        if (this.aircraftModel.model.weightclass === 'H') {
+            dataBlockRowOne = `${this.aircraftModel.callsign}  ${this.aircraftModel.model.weightclass}`;
+        }
+
+        return dataBlockRowOne;
+    }
+
+    /**
+     * Generate a string to be used for the second row of a datablock
+     *
+     * @for RadarTargetModel
+     * @method buildDataBlockRowTwo
+     * @returns {string}
+     */
+    buildDataBlockRowTwo() {
+        const aircraftAltitude = round(this.aircraftModel.altitude * 0.01);
+        const aircraftSpeed = round(this.aircraftModel.groundSpeed * 0.1);
+
+        return `${leftPad(aircraftAltitude, 3)} ${leftPad(aircraftSpeed, 2)}`;
+    }
+
+    /**
+     * Generate a string to be used for the second row of a datablock
+     * when the timeshare section is active
+     *
+     * @for RadarTargetModel
+     * @method buildDataBlockRowTwoTimeshare
+     * @returns {string}
+     */
+    buildDataBlockRowTwoTimeshare() {
```

Comment:
So there's `.buildDataBlockRowTwo()`, which calls `.buildDataBlockRowTwoTimeshare()`, making it seem like we're only "showing the timeshare" when the latter is called, which is an odd implication. For clarity, the second line _itself_ is the "timeshare", and it has a primary set of data and a secondary set, and the second row displays each of them in turn. You have it all working correctly, but I would suggest something more like `.buildDataBlockRowTwoWithPrimaryInfo()` and `.buildDataBlockRowTwoWithSecondaryInfo()` or similar. That way, it's more clear that the second row will be popluated by one or the other of these functions, without adding the inaccurate implication that the secondary info is the "timeshare".

Reply:
I figured I jacked up the words here.  done